daemon/cmd/daemon_main.go

```diff
@@ -1050,6 +1050,9 @@ func initializeFlags() {
 	flags.Bool(option.EnableK8sTerminatingEndpoint, true, "Enable auto-detect of terminating endpoint condition")
 	option.BindEnv(option.EnableK8sTerminatingEndpoint)
 
+	flags.Int(option.TCFilterPrio, 1, "Priority of tc bpf filter")
```

Comment:
Nit: `tc bpf` -> `TC BPF`? Although `TC` doesn't have a consistent casing in the wild, but let's use caps for `BPF` at least.